subprojects/build-scan-performance/src/performanceTest/groovy/org/gradle/performance/BuildScanPluginPerformanceTest.groovy

```diff
@@ -72,7 +72,7 @@ class BuildScanPluginPerformanceTest extends Specification {
         }
     }
 
-    def "build scan plugin comparison"() {
+    def "large java project with and without build scan"() {
```

Comment:
This is changed because `IndexPageGenerator` uses test name as unique id (scenario name).